### PR TITLE
Hamming: Description.md, language simplification

### PR DESCRIPTION
**Pull request by iHiD**
With reference to https://github.com/exercism/exercism/issues/4155#issuecomment-410477404 I propose making the language of this exercise easier for non-english speakers.

### CODE REVIEW

File: exercises/hamming/description.md [hunk 1]
```diff
@@ -1,31 +1,18 @@
...
-It is found by comparing two DNA strands and counting how many of the
-nucleotides are different from their equivalent in the other string.
+We read DNA using the letters C,A,G and T. Two strands might look like this:
 
     GAGCCTACTAACGGGAT
     CATCGTAATGACGGCCT
     ^ ^ ^  ^ ^    ^^
 
-The Hamming distance between these two DNA strands is 7.
+The have 7 differences, and therefore the Hamming Difference is 7.
```
Comment (rpottsoh) [inline exercises/hamming/description.md]: The => They

**Comment by iHiD** [comment]
@rpottsoh Updated. Thanks :)

**Comment by rpottsoh** [comment]
Nice catch @petertseng.  I think it should be hamming distance.  Search for hamming difference yields hamming distance....

**Comment by iHiD** [comment]
Happy to change it to "hamming distance" everywhere.

**Comment by ErikSchierboom** [comment]
> Happy to change it to "hamming distance" everywhere.

I'm also in favor of using "hamming distance".

**Comment by iHiD** [comment]
I've added another commit (we can squash on merge).

**Comment by iHiD** [comment]
Great. Thanks, all.

**Pull request merged**